### ISSUE TITLE
is this project alive?

### ISSUE THREAD

**Issue by jafara** [Carthage/Carthage#2925]
no releases from last year, is it supported or should i move to or spm?

**Comment by codeslubber** [comment]
Never mind releases, our build now takes an indeterminate amount of time because this thing hangs. Every single time.. seriously one of the worst tool breakdowns ever. Do NOT start using this unless the only other alternative is a year on an aleutian island scrounging for rat meat.

**Comment by 1oo7** [comment]
Well, the PRs have gotten some code reviews, but no merges in awhile. Not clear why.

**Comment by evandcoleman** [comment]
Any updates here? Are you in need of new maintainers? I'd be happy to volunteer my time.

**Comment by tmspzz** [comment]
@evandcoleman help is always appreciated, especially in triaging new issue

**Comment by mozeryansky** [comment]
@tmspzz I'm waiting on the the issues that have been fixed to be released. When could we expect that? We're stuck using an old Xcode because Carthage isn't supporting prebuilt binaries.

**Comment by michallaskowski** [comment]
I checked out the code and I am trying to add at least support for binary xcframeworks, based on open PRs from @tmspzz (thanks a lot for that work), cause it seems in the full scope of xcframework support they will not be merged soon. And I could use some xcframeworks for Unity and Kotlin Mutliplatform generated binaries. 

I always like to test my code, so I wanted to check it out, and apply some unit tests. It seems Project.swift could use some breakdown, into smaller classes with clear set of responsibilities (DependencyInstaller class could be one). This is what I am struggling most with any change here, and probably something that increases the level of effort for anyone willing to contribute.

Anyway, I got the feeling over the years of using Carthage, and reading through issues here, that changes are not always really welcome. Maybe this is just a feeling, maybe things changed, but if I am going to work on it, refactor, improve a bit in terms of SOLID, commit my personal time, are those changes going to be approved?

**Comment by tmspzz** [comment]
@michallaskowski changes are welcome, however since I'm "in charge" of that PR if you want to continue I can give you some pointers on the direction to take. You can contact me [on twitter](https://twitter.com/tmpz) and we'll figure out how to sync.
